domain/user-scope.js

```diff
@@ -35,31 +35,31 @@ class UserScope {
     }
 
     isRoleScope() {
-        return this.scopes?.some(scope => scope?.scope === SCOPES.ROLE);
+        return this.scopes?.some(scope => scope?.scope === SCOPES.ROLE || scope === SCOPES.ROLE);
```

Comment:
Are we changing data structure for scope? Currently, it wont work.

Reply:
I see. We need to make a unique type for Prisma. Do you think it affects another API? Or, is this unique data you have to store?

Reply:
Okay.